Assets/LeapMotion/Core/Scripts/Testing/LeapTestBase.cs

```diff
@@ -31,14 +31,32 @@ public abstract class LeapTestBase {
     /// the testing object by overriding this method.
     /// </summary>
     protected virtual void InitTest(string objectName) {
+      testObj = LoadObject(objectName);
+    }
+
+    #region Spawn Utilities
+
+    protected T Spawn<T>(T original, Vector3 position) where T : MonoBehaviour {
```

Comment:
It would be nice to relax the generic constraint so that T can just be of type UnityEngine.Object instead of UnityEngine.MonoBehaviour.  This is the same generic constraint that Instantiate<T> has, which allows you to not only spawn other components, but game objects as well!

Reply:
Added variants for GameObject and UnityEngine.Object to cover those uses